pinot-controller/src/main/java/com/linkedin/pinot/controller/ControllerStarter.java

```diff
@@ -124,7 +124,7 @@ public void start() {
 
     Configuration pinotFSConfig = _config.subset(CommonConstants.Controller.PREFIX_OF_CONFIG_OF_PINOT_FS_FACTORY);
```

Comment:
Same as `PREFIX_OF_CONFIG_OF_SEGMENT_FETCHER_FACTORY`. Other controller configs in `ControllerConf` are eager to have them reunion.